apps/organizations/admin.py

```diff
@@ -0,0 +1,35 @@
+"""
+Configure the Django admin page to manage Organizations
+"""
+from django.contrib import admin
+from django.utils.translation import ugettext_lazy as _
+
+from .models import Organization
+
+
+class OrganizationAdmin(admin.ModelAdmin):
+    """
+    Admin class for the Organization model
+    """
+    list_display = ['name', 'code', 'view_in_cms']
+
+    # pylint: disable=no-self-use
+    def view_in_cms(self, organization):
+        """
+        Add a link in the admin list view to go to the page of the organization in the CMS
+        """
+        # If the organization is linked to a page on the CMS, we want to display a link that
+        # opens a new tab and displays this page.
+        page = organization.get_page()
+        if page:
+            return '<a href="{url:s}" target="_blank">{anchor!s}</a>'.format(
```

Comment:
It's a ui decision, I would totally assume
Users of Django admin are us, the dev team
When using keyboard I prefer to close a tab, than hit backspace, because some widget could have take focus, and my right hand is still on the mouse
you ?

Reply:
I usually CTR+Click to open the link in a new tab and not having to deal with my current tab history. But it's my choice, my way of organizing the way I browse/work. Nobody forced me to do it that way.

Reply:
I have no strong opinion here. 
I always use "target=_blank" for links inside django admin, because it is a tool that people use all day long and they most probably don't want you to mess with this tab. 
My feeling is that most websites that are tools work like this (gmail for example)

Reply:
Let's ask our users then!

Reply:
And make this issue non-blocking by declaring an issue to push further the discussion?